devtools/conda-envs/environment.yaml

```diff
@@ -0,0 +1,12 @@
+name: membrane-curvature
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python
+  - pip
+  - numpy
+
+    # Pip-only installs
+  - pip:
+    - MDAnalysis=>2.0.0b0
```

Comment:
Might want to check that this works, I think I had it the wrong way around in my comment yesterday i.e. it probably should be `>=` instead of `=>`.

Reply:
Thanks so much for catching this!